tests/core/test_index_utils.py

```diff
@@ -15,7 +15,7 @@
 
 
 def test_tf_device_name_serialize_and_deserialize():
-    import tensorflow as tf
+    import tensorflow.compat.v1 as tf
```

Comment:
shouldn't this go to TF directory?

Reply:
yes, it should. it's hidden and not failing CI because it comes under the ignore list in tests.sh. should i make the changes in this PR or in a separate PR?